frontend/src/lifetime/input.ts

```diff
@@ -43,8 +43,9 @@ export function createInputManager(editor: EditorState, container: HTMLElement,
 
 		// Don't redirect user input from text entry into HTML elements
 		const { target } = e;
-		if (key !== "escape" && !(key === "enter" && e.ctrlKey) && target instanceof HTMLElement && (target.nodeName === "INPUT" || target.nodeName === "TEXTAREA" || target.isContentEditable))
+		if (key !== "escape" && !(key === "enter" && e.ctrlKey) && target instanceof HTMLElement && (target.nodeName === "INPUT" || target.nodeName === "TEXTAREA" || target.isContentEditable)) {
```

Comment:
Is it possible to put the condition on multiple lines?

Reply:
I second this, this one is a little too long.